app/form_models/publishers/job_listing/extend_deadline_form.rb

```diff
@@ -12,6 +12,7 @@ class Publishers::JobListing::ExtendDeadlineForm < BaseForm
   validates :earliest_start_date, presence: true, date: { on_or_after: :today, on_or_before: :far_future, after: :expires_at, before: :latest_start_date }, if: -> { start_date_type == "date_range" }
   validates :latest_start_date, presence: true, date: { on_or_after: :today, on_or_before: :far_future, after: :earliest_start_date }, if: -> { start_date_type == "date_range" }
   validates :other_start_date_details, presence: true, if: -> { start_date_type == "other" }
+  validates :extension_reason, inclusion: { in: Vacancy.extension_reasons.keys }
```

Comment:
do we need validation on other_extension_reason_details is extension_reason is "other_extension_reason"?